C/common/logger.cpp

```diff
@@ -32,15 +32,26 @@ Logger *Logger::instance = 0;
 
 Logger::Logger(const string& application)
 {
-	m_app_name = new string(application);
-	openlog(m_app_name->c_str(), LOG_PID|LOG_CONS, LOG_USER);
+static char ident[80];
+
+	/* Prepend "Fledge " is not already there.
+	 */
+	if (application.substr(0, strlen("Fledge")).compare("Fledge") != 0)
+	{
+		string logname = string("Fledge ") + application.c_str();
+		snprintf(ident, sizeof(ident), "Fledge %s", logname.c_str());
```

Comment:
It is appending **Fledge** twice
On **Raspberry Pi** I see the logs atleast, but I see different behavior on **UB** environment
```
Apr 24 13:36:47 aj Fledge Fledge T HING[5514]: WARNING: ThingSpeak Nothing to send
Apr 24 13:35:52 aj Fledge Fledge bench[5139]: INFO: Configuration change notification for bench
Apr 24 13:38:56 aj Fledge Fledge bench #1[5440]: INFO: Configuration change notification for bench #1
```